### ISSUE TITLE
add relational diagrams for example datasets

### ISSUE THREAD

**Issue by 7yl4r** [ioos/bio_data_guide#57]
I have previously used quickDBD to create diagrams of how the occurence, MoF, event, etc files/tables connect with each other. I find these views tremendously useful and would like to encourage including them for the example datasets. 

Then again... does anyone else find database diagrams useful or is it just me?

### bare-minimum example (tier 0):

![image](https://user-images.githubusercontent.com/1051390/133847558-02d21e09-9ed6-4bd3-9aa2-204502dec9ac.png)

### more detailed example: 

![image](https://user-images.githubusercontent.com/1051390/133847595-53f6fda9-e182-4a12-ba08-157de3f3cb31.png)

**Comment by MathewBiddle** [comment]
I think those are very helpful!

**Comment by MathewBiddle** [comment]
it would be interesting to put that subsample example in this visual https://github.com/ioos/bio_data_guide/issues/41

**Comment by albenson-usgs** [comment]
Yes very helpful. Only question for me is if separating event MOFs from occurrence MOFs will give the impression they are separate files?

**Comment by 7yl4r** [comment]
When I made this I think I was thinking that they were separate files.

So each row in the MoF table has both an `eventID` and `measurementID` column and only one should ever be filled out?

**Comment by albenson-usgs** [comment]
All of them will have an eventID but only the ones that are linked to occurrences will have occurrenceIDs. See the table at the very bottom [here](https://ioos.github.io/ioos_code_lab/content/Code%20Gallery/data_management_notebooks/2020-12-08-DataToDwC.html).

**Comment by albenson-usgs** [comment]
I've never used measurementID.

**Comment by 7yl4r** [comment]
> All of them will have an eventID but only the ones that are linked to occurrences will have occurrenceIDs. See the table at the very bottom [here](https://ioos.github.io/ioos_code_lab/content/Code%20Gallery/data_management_notebooks/2020-12-08-DataToDwC.html).

Ah, that makes perfect sense. I will update the diagram before posting it elsewhere.

**Comment by Br-Johnson** [comment]
There's also a great R package for this called `dm` https://cynkra.github.io/dm/

I personally find them very helpful though they made need some explanation for those unfamiliar with them.
